Library/Formula/ccnet.rb

```diff
@@ -0,0 +1,45 @@
+class Ccnet < Formula
+  homepage "http://www.seafile.com/"
+  url "https://github.com/haiwen/ccnet/archive/v4.1.3.tar.gz"
+  sha1 "bbf052d1f37e3824c32088654dd8ad49aec8e019"
+
+  head "https://github.com/haiwen/ccnet.git"
+
+  #[FIX] homebrew autotools path
+  patch :p1 do
+    url "https://github.com/Chilledheart/ccnet/commit/48610f08.diff"
+    sha1 "af5a53ffb9fca0d4918b9d2ef1afd54959ecf186"
+  end
+
+  #[FIX] openssl build
```

Comment:
Can you please submit these patches to the upstream developers of this project and add a link to the upstream patch submission and explanation of why the patch is needed in a comment in the formula file. Thanks!